apps/core/scenes/results/Results.js

```diff
@@ -54,6 +54,9 @@ type Props = {|
   +sortBy: string,
   +layout: number,
   +limit: number,
+  +nightsInDestinationFrom: string,
```

Comment:
Does it needs to be a string? I see that you are then parsing it to number on multiple places. And same prop in `ResultsScreen` already have `number` type, so it will be nice to unify it.
Same with `nightsInDestinationTo`.

Reply:
Done

Reply:
Done